notebooks/106-auto-device/106-auto-device.ipynb

```diff
@@ -25,9 +25,8 @@
    "metadata": {},
```

Comment:
<p>it's not saved to <code>../model/ir_model</code>, but to <code>model/ir_model</code>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/865/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
done